nextpnr.dockerfile

```diff
@@ -106,6 +106,33 @@ RUN cd /tmp/nextpnr/build \
  && make -j $(nproc) \
  && make DESTDIR=/opt/nextpnr install
 
+
+#---
+
+FROM build AS build-gowin
+RUN mkdir -p /tmp/nextpnr/build \
+ && apt-get update -qq \
+ && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends python3-setuptools python3-pip  \
+ && pip3 install apycula \
+ && cd /tmp/nextpnr \
+ && curl -fsSL https://codeload.github.com/YosysHQ/nextpnr/tar.gz/master | tar xzf - --strip-components=1 \
+ && cd build \
+ && cmake .. \
+   -DARCH=gowin \
+   -DBUILD_GUI=OFF \
+   -DBUILD_PYTHON=ON \
+   -DUSE_OPENMP=ON \
+ && make -j $(nproc) \
+ && make DESTDIR=/opt/nextpnr install
+
+#---
+
+FROM base AS gowin
```

Comment:
There should be two stages here:

```
FROM base AS gowin
```

and

```
FROM gowin AS apicula
COPY --from=hdlc/pkg:apicula /apicula /
```

The first one would contain artifacts of nextpnr-gowin only; and the second one would include apicula/apycula too.